docs/content/using/using_hostdefinition.md

```diff
@@ -0,0 +1,73 @@
+     
+          $> kubectl get hostdefinition | grep <hostname> | wc -l
+     
+     The output should be `0`.
+3. Redeploy the CSI node pod on the relevant node.
+
+     The host definer handles all of the new host definitions.
+        
+4. Verify that the `hostdefinition` is in the _Ready_ phase.
+
+    ```
+    $> kubectl get hostdefinition
+    NAME                     AGE    PHASE   STORAGE               NODE
+    <host_definition_name1>  102m   Ready   <management_address>  <node_name1>
+    <host_definition_name2>  102m   Ready   <management_address>  <node_name2>
+    ```
+
+## Updating the `HostDefiner` custom resource
+
+Some of the parameters within the `HostDefiner` custom resource are configurable. Use this information to help decide whether the parameters for your storage system need to be updated.
+    
+|Field|Description|
+|---------|--------|
+|prefix|Adds a prefix to the hosts defined by the host definer.|
+|connectivity|Selects the connectivity type for the host ports.<br>Possible input values are:<br>- `nvme` for use with NVME over Fibre Channel connectivity<br>- `fc` for use with Fibre Channel over SCSI connectivity<br>- `iscsi` for use with iSCSI connectivity<br><br>By default, this field is blank and the host definer selects the first of available connectivity types on the storage system, according to the following hierarchy: NVMe, FC, iSCSI.|
+|allowDelete|Defines whether the host definer is allowed to delete host definitions.<br>Input values are `true` or `false`.<br>The default value is `true`.|
+|dynamicNodeLabeling|Defines whether the nodes that run our CSI node pod are dynamically labelled or if the user must create the `hostdefiner.block.csi.ibm.com/managed-by=true` label on each relevant node. In addition, this label allows the host definer to find all nodes with this label and manages their storage definition.<br>Input values are `true` or `false`.<br>The default value is `false`, where the user must create each label.|
+
+## Blocking a specific node definition from being deleted
+
+In order to block a specific host definition from being deleted by the host definer, you can add the following label to the node: `hostdefiner.block.csi.ibm.com/avoid-deletion=true`.
+
+This works on a per node basis, where the `allowDelete` parameter definition in the `csi_v1_hostdefiner.yaml` is for all cluster nodes.
+
+## Changing host connectivity
+
+When node connectivity changes take place, use the following procedure to redefine host connectivity:
+
+1. Undeploy the CSI node pod from the relevant node that the `HostDefinition` is a part of.
+2. Verify that all `HostDefinition` instances of the node are deleted.
+     
+          $> kubectl get hostdefinition | grep <hostname> | wc -l
+     
+     The output should be `0`.
+3. From the host, change the host connectivity type.
+4. Redeploy the CSI node pod on the relevant node.
+
+     The host definer handles all of the new host definitions.
```

Comment:
```suggestion
4. Redeploy the CSI node pod on the relevant node.

     The host definer handles the new creation of the new host definition.
```
if the node is singular, so does its definition on storage (currently)

Reply:
updated

Reply:
I don't see any change.
please always ensure the changes appear under the `Files changed` tab, or at least that the conversation has an `Outdated` label at the top, as can be seen in other conversations:
![image](https://user-images.githubusercontent.com/57443811/176901807-76171d33-b03f-4d48-9c85-4938a11a8b47.png)
**before** writing "done"/"updated"/etc...

Reply:
please check again. I only write updated after I push to github

Reply:
it was pushed [9 hours ago](https://github.com/IBM/ibm-block-csi-driver/commit/7ea6a193ddd92c10693076b3ef8d52e54e39c86e#diff-b497e2484be1ca3b21e64c4f5ed6cfff1afecf6ef363b6307e272ecb5751e8c5)